### ISSUE TITLE
Anko Layouts preview

### ISSUE THREAD

**Issue by chendongMarch** [Kotlin/anko#470]
I can not find View -> Tool Windows -> Anko Preview

Rebuild and clean is not work.

Android Studio 3.0 Beta 5
Kotlin 1.1.4
Anko 0.10.1

```
class TestLayoutsActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        TestLayoutsActivityUI().setContentView(this)
    }
}

class TestLayoutsActivityUI : AnkoComponent<TestLayoutsActivity> {
    override fun createView(ui: AnkoContext<TestLayoutsActivity>): View = with(ui) {
        verticalLayout {
            backgroundColor = 0xff.gray.opaque
            val textView = textView("textview")
        }
    }
}
```

**Comment by luc743** [comment]
Hi, there is a new commit which correct bugs for AS 3.0. I dont know how to compile and make a jar plugin file ? Can anybody help me ?

**Comment by davidbilik** [comment]
You have instructions for building Anko here https://github.com/Kotlin/anko/wiki/Building-Anko. I've tried it just now, succesfully generated jar file for Idea plugin and preview is working <3 Unfortunately I cannot make it to work with subclasses of my own BaseAnkoComponent :( But better than nothing

**Comment by luc743** [comment]
Hey, thanks you, you could give me your jar idea plugin ?

**Comment by davidbilik** [comment]
[preview.jar.zip](https://github.com/Kotlin/anko/files/1314148/preview.jar.zip)
But keep in mind that JB has probably reasons why they didnt release it yet, so dont report bugs until they do. You have to extract the zip

**Comment by luc743** [comment]
Thanks you, yes i keep that in mind, but i tried everything and nothing worked and we really need that plugin because we working on a project which all views are written in Anko and we loose too much time to test views

**Comment by davidbilik** [comment]
Honestly I dont know if that plugin will help you, because you must build your project to display layout in preview correctly

**Comment by luc743** [comment]
You are right ..

**Comment by JDDJJ** [comment]
will this plugin update? there are so many errors in 0.10.1

**Comment by 4u7** [comment]
Fixed with c685e07256673f144d788c51865c6a42e13da120
